packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/simulation/ProgressState.java

```diff
@@ -3,6 +3,8 @@
 import com.fasterxml.jackson.annotation.JsonAlias;
 
 public enum ProgressState {
+	@JsonAlias("cancelling")
```

Comment:
I think that this will require a DB migration @kbirk ?

Reply:
Is the DB migration something that can be done easily? I'm not quite familiar with the process.

Reply:
I placed the new enum value to the end so that it won't shuffle the existing value (e.g make sure `cancelled` was 0 and 0 is not replaced by `cancelling`) which might cause the issue with the `state` in `NotificationEvent`. Aside from that It doesn't look like we need to worry about the migration.